AirCasting/CreateSessionViews/ConfirmCreatingSessionView.swift

```diff
@@ -32,24 +43,33 @@ struct ConfirmCreatingSessionView: View {
             GoogleMapView(pathPoints: [], values: [], isMyLocationEnabled: true)
                         
             Button(action: {
-                sessionContext.setupAB()
-                didStartRecordingSession = true
+                if (sessionContext.deviceType == DeviceType.MIC) {
+                    sessionContext.startMicrophoneSession(microphoneManager: microphoneManager)
+                } else {
+                    sessionContext.setupAB()
+                }
+                self.isActive = true
             }, label: {
                 Text("Start recording")
                     .bold()
             })
+            .background(
```

Comment:
Yes, I know, that's why I left a todo note and created trello ticket for that. It is related to all of the sessions, not only microphone, so I thought it could be done separately.